app/client/src/PluginActionEditor/components/PluginActionResponse/components/Response/Response.tsx

```
@@ -0,0 +1,353 @@
+                  actionResponse.body &&
+                  action.pluginType === PluginType.DB && (
+                    <div data-testid="t--response-error">
+                      {actionResponse.body}
+                    </div>
+                  )
+                ))}
+              <LogHelper
+                logType={LOG_TYPE.ACTION_EXECUTION_ERROR}
+                name="PluginExecutionError"
+                pluginErrorDetails={
+                  actionResponse && actionResponse.pluginErrorDetails
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)